Sources/web3swift/Web3/Web3+Resolver.swift

```diff
@@ -17,6 +17,7 @@ public class PolicyResolver {
     }
 
     public func resolveAll(for tx: inout CodableTransaction, with policies: Policies = .auto) async throws {
+        // FIXME: This should throw an appropriate error, since this condition is critical to whole function tast
```

Comment:
I'd add an `else` case with a simple `throw Web3Error.valueError` for now.

You can temporarily modify `case valueError` to accept an optional description: `case valueError(desc: String? = nil)`. It won't break the other parts of the code but will give you an option to specify an error message here.
Later it can be updated to `case valueError(desc: String)`.